content/en/getting_started/database_monitoring/_index.md

```diff
@@ -0,0 +1,123 @@
+```
+
+4. Set the environment variable `DD_API_KEY` to your Datadog API key:
+```
+export DD_API_KEY=<API_KEY>
+```
+
+5. Start the application:
+```
+make postgres
+```
+
+The command will continue to run until you stop it by pressing Ctrl + C.
+
+## Identify an expensive query
+
+Which query consumes the most database time? To find out, use the Query Metrics view.
+
+1. Navigate to Database Monitoring by clicking **APM > Databases** in the UI. You will see the Query Metrics view.
+
+2. Sort the Normalized Query table by **Percent time** to see the query that the database spends the most time executing.
+
+The query that consumes the most database time will appear on the first line.
+
+{{< img src="database_monitoring/dbm_qm_sort_time.png" alt="Normalized queries sorted by percent time" style="width:100%;">}}
+
+## Troubleshoot a slow query
+
+In addition to identifying slow queries, Datadog Database Monitoring can help you diagnose them. A query's Explain Plan describes the steps that the database takes to resolve the query. View an Explain Plan by clicking on a sample in the Query Samples view.
+
+Navigate to the Query Samples view within Database Monitoring by clicking **[APM > Databases][5]**, and selecting the **Query Samples** tab in the UI.
+
+Sort the Normalized Query table by **Duration**.
+
+{{< img src="database_monitoring/dbm_qs_sort_duration.png" alt="Normalized query samples sorted by duration" style="width:100%;">}}
+
+Find a query in the table with data in the **Explain Plan** column, and click on it to open the Sample Details page. This Explain Plan at the bottom of Sample Details shows that the query requires an _Index Scan_.
+
+{{< img src="database_monitoring/dbm_qs_explain_plan.png" alt="Query explain plan showing Index Scan" style="width:100%;">}}
+
+## View changes in query volume
+
+To understand the health and performance of your databases at a glance, add Datadog Database Monitoring metrics to a dashboard.
+
+For example, you can see the absolute change in query volume in the past hour by adding a **Change** widget to track a query count metric.
+
+1. Select **Dashboards > New Dashboard** in the UI.
+
+2. Enter a name for your dashboard. Click the **New Dashboard** button to go to your new dashboard.
+
+2. To add content to your dashboard, click **Add Widgets**.
+
+3. In the widget carousel, select the **Change** widget.
+
+4. Select `postgresql.queries.count` in the **Metric** dropdown. This metric counts the number of queries sent to a PostgreSQL database.
+
+5. Select `host` in the **Break it down by** dropdown so the widget aggregates queries by host.
+
+{{< img src="database_monitoring/dashboard_change_postgres.png" alt="Configure change widget for postgres queries metric" style="width:100%;">}}
+
+7. Click the **Save** button. The dashboard shows your new widget.
+
+{{< img src="database_monitoring/dashboard_change_widget.png" alt="Change widget showing query count" style="width:100%;">}}
+
```

Comment:
Here, let's wrap up by telling them that there is also an out-of-the-box dashboard for Postgres and MySQL that they can access through the Dashboards button on the DBM page, and which they can clone and modify to suit their needs.